verifier-sdk/src/state.rs

```diff
@@ -89,6 +89,35 @@ impl<
     }
 }
 
+#[derive(AnchorDeserialize, AnchorSerialize)]
+pub struct PathNode {
+    pub node: [u8; 32],
+    pub index: u32,
+}
+
+/// Event containing the Merkle path of the given
+/// [`StateMerkleTree`](light_merkle_tree_program::state::StateMerkleTree)
+/// change. Indexers can use this type of events to re-build a non-sparse
+/// version of state Merkle tree.
+#[derive(AnchorDeserialize, AnchorSerialize)]
+pub struct ChangelogEvent {
```

Comment:
I noticed that with parallel Merkle trees we will need a vector of changelog events because we will need to emit such event for every Merkle tree that we update.
Can you already add that here?
For example:
```
#[derive(AnchorDeserialize, AnchorSerialize)]
pub struct ChangelogVec {
  pub changelogs: Vec<ChangelgEvent>,
  }

```